documentation/kernel-semantics.tex

```diff
@@ -3152,12 +3175,29 @@ \subsubsection{Function Declaration}
 \subsection{Event Loop}
 \label{subsec:eventloop}
 
+\dart{} supports asynchronous execution of code with async functions and await expressions.
+We introduce an event loop to define the behaviour of asynchronous code.
+We denote the event loop with $\eventloop$.
+$\eventloop$ represents a list of event continuations, elements of $\dncont$.
+\[
+    \eventloop \in \mlist{\dncont}
+\]
+
+The event loop is a list and it supports the operations defined in Section~\ref{subsubsec:semantic-list}.
+
+The event loop appears on the left and the right-hand side of all configurations, therefore we omit it from rules where the same queue just passed from one state to the other.
```

Comment:
I removed this text and added to the list of things to be done the modification of configurations to account for the event loop.

We could add a global stack trace and mention it only when calls and return statements are executed. It might simplify the notations and look nicer.